content/en/cloud_cost_management/azure.md

```diff
@@ -158,7 +158,73 @@ You can visualize your ingested data using the following cost types:
+| `billingprofilename` | The name of the Enterprise Agreement enrollment. |
+| `chargetype` | The type of charge covering the line item: `Usage`, `Purchase`, or `Refund`. |
+| `consumedservice` | The name of the service the line item is associated with. |
+| `costcenter` | The cost center defined for the subscription for tracking costs. |
+| `costinbillingcurrency` | The cost in the billing currency before credits or taxes. |
+| `costinpricingcurrency` | The cost in the pricing currency before credits or taxes. |
+| `currency` | The currency associated with the billing account. |
+| `date` | The usage or purchase date of the charge. |
+| `effectiveprice` | The blended unit price for the period. Blended prices average out any fluctuations in the unit price, like graduated tiering, which lowers the price as quantity increases. |
+| `exchangeratedate` | The date the exchange rate was established. |
+| `exchangeratepricingtobilling` | The exchange rate used to convert the cost in the pricing currency to the billing currency. |
+| `frequency` | Indicates whether a charge is expected to repeat. Charges can either happen once (`OneTime`), repeat on a monthly or yearly basis (`Recurring`), or be based on usage (`Usage`) |
+| `InvoiceId` | The unique document ID listed on the invoice PDF. |
+| `invoicesectionid` | The ID of the MCA invoice section. |
+| `invoicesectionname` | The name of the EA department. |
+| `isazurecrediteligible` | true if the charge is eligible to be paid for using Azure credits. |
+| `location` | The data center location where the resource is running. |
+| `metercategory` | The top level service that this usage belongs to (such as `Networking`). |
+| `meterid` | The unique ID for the meter. |
+| `metername` | The usage details of the line item (such as `L8s v2` or `General Purpose Data Stored`). |
+| `meterregion` | The data center location for the services priced based on location (such as `West US 2`). Use `resourcelocation` to see location data without N/A. |
+| `metersubcategory` | The name of the meter subclassification category (such as `General Purpose - Storage`). Use `metername` or `metercategory` to see top-level classification without N/A. |
+| `offerid` | The name of the offer purchased. |
+| `partnumber` | The ID used to get specific meter pricing. |
+| `planname` | The marketplace plan name if purchased through marketplace. |
+| `PreviousInvoiceId` | Reference to an original invoice if this line item is a refund. |
+| `PricingCurrency` | The currency used when rating based on negotiated prices. |
+| `pricingmodel` | The type of usage (such as `Reservation`). |
+| `ProductId` | The identifier for a specific Azure product. |
+| `productname` | The name of the Azure product at a granular level, such as VM or disk type and region. |
+| `productorderid` | The ID for the product order. Use `productname` to see top level product information without N/A. |
```

Comment:
It seems like N/A is a value, so I would wrap it in backticks wherever it shows up.